### ISSUE TITLE
Android Security Alert: Another app on your phone may be trying to...

### ISSUE THREAD

**Issue by vikrama** [dropbox/dropbox-sdk-java#293]
Using the SDK in a debug build in Android emulator, dropbox.AuthActivity errors out with a Security Alert "There might be more than one app trying to handle....". 

I see that it is very similar to : https://github.com/dropbox/dropbox-sdk-java/issues/153 and the error is https://github.com/dropbox/dropbox-sdk-java/blob/5f47303107da2813f696b520ce976b73d0ba7847/src/main/java/com/dropbox/core/android/AuthActivity.java#L289

1. My app is the only non-default app in the emulator.
2. I have verified that there are no multiple declarations of dropbox.AuthActivity in AndroidManifest files.
3. Have tried uninstalling and re-installing the debug build multiple times.

Any idea what's going on?

**Comment by vikrama** [comment]
With the advent of App Bundle, can the dropbox.AuthActivity declaration continue to be in a module's manifest or should it be in the base module's manifest? For some reason, when AuthActivity starts, it's finding 2 instances of it under my app's packagename.

**Comment by greg-db** [comment]
Can you try restarting the emulator's OS, if you haven't already? While not typically necessary, in some cases we've heard that a restart is required for the system to reflect changes to these.

**Comment by vikrama** [comment]
Yes, behavior continues after a cold start and even after restarting the OS.

**Comment by greg-db** [comment]
Thanks for checking that. I'm not sure off hand what may be causing this then, so we'll have to look into it. (It may be related to using an App Bundle, but I don't know if the SDK has been tested with that in particular. I believe it predates App Bundles.)

To help us reproduce this, would you be able to share am AndroidManifest.xml file(s) (or a sample project, if possible), that produces this issue? Thanks in advance!

**Comment by Sardarwaqas800** [comment]
Clear all issue
